src/helpers/cliUtils.js

```diff
@@ -135,6 +135,7 @@ async function app(Client, fromDate, toDate, pathToConfig, pathToRunLogs, debug,
     if (debug) logger.level = 'debug';
     checkLogFile(pathToRunLogs);
     const config = getConfig(pathToConfig);
+    if (typeof config.commonExtractorArgs === 'undefined') {config.commonExtractorArgs = {}};
```

Comment:
DM1: I like adding this check just in case, but would you mind changing this check to the following?
```
if (!config.commonExtractorArgs) config.commonExtractorArgs = {};
```
The `typeof` check is unnecessary since we can be confident the `config` exists from the above statement (or would have already thrown an error and halted prior to this), and this is a simplification of the above since this will evaluate to a falsey value if it doesn't exist. Also, for style reasons, you don't need to have the brackets around the execution block of a conditional when it's on a single line. :)

Reply:
DM1: I gotcha! Just made the change, thanks for the heads up :).